ups_modul/inout/inout.py

```diff
@@ -171,11 +170,12 @@ def loop():
 # weird stuff
 if __name__ == "__main__":
     setup_inout()
-    connected = False
-    while connected == False:
+    connected = "OFF"
```

Comment:
comparing strings is O(n) time. Consider using boolean or enum instead.

Reply:
changed back to boolean and adding print when connection is up